rerun_py/rerun_sdk/rerun/components/line_strip2d_ext.py

```diff
@@ -42,13 +43,36 @@ def native_to_pa_array_override(data: LineStrip2DArrayLike, data_type: pa.DataTy
         elif isinstance(data, Sequence):
             if len(data) == 0:
                 inners = []
-            elif isinstance(data[0], np.ndarray):
-                inners = [Vec2DBatch(datum).as_arrow_array().storage for datum in data]  # type: ignore[arg-type]
-            elif isinstance(data[0], LineStrip2D):
-                inners = [Vec2DBatch(datum.points).as_arrow_array().storage for datum in data]  # type: ignore[union-attr]
             else:
-                inners = [Vec2DBatch(datum).as_arrow_array().storage for datum in data]  # type: ignore[arg-type]
-
+                # Is it a single strip or several?
```

Comment:
there might be some path of "just reinterpret everything as nd array and go from there" but I couldn't quite figure it out since we also allow sequences containing our own objects.